packages/mjml-core/src/MJMLRenderer.js

```diff
@@ -71,7 +71,9 @@ export default class MJMLRenderer {
       return;
     }
 
-    this.errors = MJMLValidator(this.content)
+    this.errors = (this.options.validator instanceof MJMLExtendedValidator) ?
```

Comment:
Mh, ça me semble un peu étrange le validateur devrais pas prendre un tableau d'option plutôt ?

`MJMLValidator(this.content, this.options.validator)`